deployment/README.md

```diff
@@ -98,3 +105,28 @@ If something went wrong, you can reconfigure your config files and rerun:
 docker-compose build
 docker-compose up -d
 ```
+
+#### Common issues
+- Errors about TLS issues on User cration point to something amiss in the mail server configuration.
+- Errors about unknown client point to ClientID mismatch between ``api.json`` and ``frontend.json``
+
+### Updates
+
+The above was designed to simplify updates, to upgrade to the latest 
+portal version, backup your ``$ROOT/data`` directory. Then run the upgrade as follows:
+
+```
+pushd source
+git pull --recurse-submodules
+popd
+docker-compose build
+# if the above was successful the following will replace the updated containers with their latest versions
+docker-compose up -d
+```
+Most of the time this will be all that is required to update
```

Comment:
Thank you for clarifying. I changed the section to not give wrong advice on updates instead. I kept it in because I was reasoning that others may notice the pristine sources folder and deduce that it eases updates.

I'd also not complain if you'd rather keep this out of here entirely for the time being.